src/cmd-buildextend-metal

```diff
@@ -35,7 +35,8 @@ EOF
 
 # Parse options
 hostkey=/srv/src/config/secex-hostkey
```

Comment:
Isn't this default never used now? Can we empty it out? Then we should be able to get rid of `genprotimgvm_path`.

Reply:
`hostkey_path` is still used on local development builds. Basically we only use `genprotimgvm_path` in the CI.

Reply:
I've rebased the PR and removed `genprotimg_path`